jerryscript.js

```diff
@@ -79,19 +79,43 @@ var asyncTestHelperHead =
 '  asyncPassed = true;\n' +
 '}\n' +
 '\n' +
-'function setTimeout(cb, time, ...cbarg) {\n' +
+'function setTimeout(cb, time, cbarg) {\n' +
 '  if (!jobqueue[time]) {\n' +
 '    jobqueue[time] = [];\n' +
 '  }\n' +
-'  jobqueue[time].push({cb, cbarg});\n' +
+'  jobqueue[time].push({cb, cbarg, startTime: new Date().getTime(), timeout: time});\n' +
```

Comment:
```suggestion
'  jobqueue[time].push({cb, cbarg, startTime: Date.now(), timeout: time});\n' +
```
sorry, missed this one